src/ray/object_manager/object_buffer_pool.h

```diff
@@ -117,15 +117,17 @@ class ObjectBufferPool {
   /// \param chunk_index The index of the chunk.
   void AbortCreateChunk(const ObjectID &object_id, uint64_t chunk_index);
 
-  /// Seal the object associated with a create operation. This is invoked whenever
-  /// a chunk is successfully written to.
+  /// Write to a Chunk of an object.
```

Comment:
Can you also mention Seal is happening within this method? I feel like since Seal is an exposed concept to the upper layer, hiding it can confuse readers